### ISSUE TITLE
Susijusios literatūros apžvalga

### ISSUE THREAD

**Issue by gisora** [gisora/ptu8-final-project#1]
- [x] [PubMed 200k RCT: a Dataset for Sequential Sentence Classification in Medical Abstracts](https://arxiv.org/pdf/1710.06071.pdf)
- [x] [Neural Networks for Joint Sentence Classification in Medical Paper Abstracts](https://arxiv.org/pdf/1612.05251.pdf)
- [x] [The multinomial Naive Bayes classifier](https://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.MultinomialNB.html)
- [x] [scikit-learn: TfidfTransformer](https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfTransformer.html)
- [x] [scikit-learn: LabelEncoder](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html)
- [x] [scikit-learn: Pipeline](https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html)
- [x] [TensorFlow: TextVectorization](https://www.tensorflow.org/api_docs/python/tf/keras/layers/TextVectorization)
- [x] [TensorFlow: Embedding](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Embedding)
- [x] [TensorFlow: Better performance with the tf.data API](https://www.tensorflow.org/guide/data_performance)
- [x] [TensorFlow: Build TensorFlow input pipelines](https://www.tensorflow.org/guide/data)
- [x] [TensorFlow: Dataset from tensor slices](https://www.tensorflow.org/api_docs/python/tf/data/Dataset#from_tensor_slices)
- [x] [TensorFlow: Conv1D](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv1D)
- [x] [TensorFlow: GlobalAveragePooling1D](https://www.tensorflow.org/api_docs/python/tf/keras/layers/GlobalAveragePooling1D)
- [x] [TensorFlow: TensorBoard](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard)
- [x] [GloVe: Global Vectors for Word Representation](https://nlp.stanford.edu/projects/glove/)
- [x] [TFHub: Universal Sentence Encoder](https://tfhub.dev/google/universal-sentence-encoder/4)
- [x] [TensorFlow: Bidirectional](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Bidirectional)
- [x] [TensorFlow: LSTM](https://www.tensorflow.org/api_docs/python/tf/keras/layers/LSTM)
- [x] [TensorFlow: Concatenate](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Concatenate)
- [x] [TensorFlow: Dropout](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dropout)
- [ ]
